rte/mo_rte_util_array.F90

```diff
@@ -75,9 +79,28 @@ logical function any_vals_less_than_3D(array, check_value)
 
     real(wp) :: minValue
 
+#ifdef _OPENMP
```

Comment:
Here, too - will Cray eventually support `minval` and `maxval` intrinsics?

Reply:
`minval` and `maxval` are supported on the GPU. The problem here has to do with performance.

Sadly, OpenMP 4.5 - 5.0, does not get Fortran array notation and there is no equivalent to `$!acc kernel`. So the loops have to be explicit to expose the parallelism. Thus, you have to work with the scalar versions, i.e. `min` and `max`.

I avoided writing something custom, but without these customized routines. The allsky benchmark with tons of columns does not see any speed-up on the GPU.